client/js/widgets/spinner.js

```diff
@@ -16,7 +16,8 @@ class Spinner extends Widget {
       spinnerCSS: '',
       valueCSS: '',
       textColor:'#000000',
-      lineColor :'#d2d2d2'
+      lineColor :'#d2d2d2',
+      borderRadius: '50%'
```

Comment:
Good point. 50% is more accurate (any single percentage value over 50% gets reduced to 50% because nothing larger will fit). I'll change all to 50% (even though some currently are in the stylesheet as 100%.